permissions/component-jenkins-test-harness-htmlunit.yml

```diff
@@ -7,4 +7,3 @@ paths:
   - "org/jenkins-ci/main/jenkins-test-harness-htmlunit"
 developers:
   - "drulli"
-  - "@core"
```

Comment:
why remove from here? even if using CD should still be able to release from here as a backup and this is also used for security contacts